# request a machine state, ON of OFF
string req_transition
---
# reply, can be ON, OFF or ERROR
string rep_state
